.github/workflows/graph-develop.yaml

```diff
@@ -0,0 +1,62 @@
+name: Deploy Graph to develop
+
+on:
+  push:
+    branches: [develop]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16
+      - name: Install
+        run: yarn --frozen-lockfile
+      - name: Lint
+        run: yarn lint
+
+  generate-manifests:
+    runs-on: ubuntu-latest
+    needs: lint
+    steps:
+      - name: Lint
+        run: yarn generate-manifests
+
+  deploy-staging-goerli:
+    needs: generate-manifests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16
+      - name: Install
+        run: yarn --frozen-lockfile
```

Comment:
Why do we install these dependencies again? they are already installed in dependency jobs